src/App.css

```diff
@@ -5,29 +5,20 @@
   padding: 0;
   margin: 0;
 }
-header {
-  font-size: 1.5rem;
-  color:rgb(119, 119, 252);
-  font-weight: 700;
-  text-align: center;
-}
-body {
-  font-family: "Sora", sans-serif;
+.App {
   display: flex;
-  justify-content: center;
+  flex-direction: column;
```

Comment:
making this would change every page to have `flex-direction: column;`
i think you should only make this change on specific page